src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediateStatement.java

```diff
@@ -222,6 +272,10 @@ public String getCommand() {
     return this.command;
   }
 
+  public boolean isBatchedQuery() {
```

Comment:
Is this only used for testing? If yes consider adding a comment explaining.

Reply:
Yes, this is only for testing. I added a note for that.